emanifest-py/src/emanifest/client.py

```diff
@@ -681,12 +681,16 @@ def search_mtn(self, reg: bool = False, **kwargs) -> RcrainfoResponse:
         Keyword Args:
             stateCode (str): Two-letter US postal state code
             siteId (str): EPA Site ID
+            submissionType (str): FullElectronic, Hybrid, Image, DataImage5Copy
```

Comment:
Looks good, We need to update the [search services documentation](https://usepa.github.io/e-manifest/Services/Manifest/search.html) to reflect the changes.